.evergreen/config.yml

```diff
@@ -1771,7 +1773,10 @@ tasks:
         - func: "bootstrap mongo-orchestration"
           vars:
             TOPOLOGY: "sharded_cluster"
+            LOAD_BALANCER: true
         - func: "run load-balancer"
+          vars:
+            LOAD_BALANCER: true
```

Comment:
I don't think vars/LOAD_BALANCER is needed for "run load-balancer".

Reply:
Done.

Reply:
LOAD_BALANCER is not needed for run tests either is it?

Reply:
LOAD_BALANCER is only needed for run-orchestration.sh. Let's remove it here. Note we can also remove the `export LOAD_BALANCER=1` line above.